src/main/java/com/minecolonies/coremod/entity/ai/citizen/planter/EntityAIWorkPlanter.java

```diff
@@ -106,23 +142,29 @@ private IAIState farm()
             return START_WORKING;
         }
 
-        if (walkToBlock(workPos.above()))
+        if (isItemPositionAir(workPos))
+        {
+            return START_WORKING;
+        }
+
+        if (walkToBlock(workPos.getPosition().above()))
```

Comment:
I think we should rename the workPos, since its always the ground block and we keep having to do above() its kinda confusing. Maybe plantableSoilPos? or so